__tests__/app.test.js

```diff
@@ -74,6 +74,115 @@ describe(`NCNews-Server Unit Tests`, () => {
         });
 
            
+    });
+
+    // Error Handling
+    describe("Error Handling Tests", () => {
+        // GET METHOD 404 ERROR TEST
+        test("[ 404 ] Responds with a 404 error when an invalid path is specified", () => {
+          return request(app).get("/sdfhdshifsdhfsd").expect(404).then((response) => {
+              expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+            });
+        });
+        // need 500 internal server error test - find out what to do for that one
+    });
+
+    // Task 5 - get api articles articleid
+
+    describe(`GET /api/articles/:article_id`, () => {
+
+        test(`[ 200 ] Responds with an object with key article and keyval of an object with the following 7 properties: author, title, article_id, topic, body, created_at, votes`, () => {
+
+            return request(app).get(`/api/articles/1`).expect(200).then((response) => {
+                const article = response.body.article;
+
+                    expect(article).toEqual(
+                        expect.objectContaining({
```

Comment:
As we are getting a specific article here then we should expect specific values rather than `any`.